dev-tools/generate_notice.py

```diff
@@ -102,6 +134,11 @@ def create_notice(filename, beat, copyright, vendor_dirs):
 
         if 'vendor' in dirs:
             vendor_dirs.append(os.path.join(root, 'vendor'))
+            dirs.remove('vendor')   # don't walk down into sub-vendors
```

Comment:
Right, "vendor in vendor" is what it's being excluded here, but those deps are still found on the walk from line 42.